tests/integration/create_cluster/complex/expected-v1alpha2.yaml

```diff
@@ -11,7 +11,6 @@ spec:
   channel: stable
   cloudProvider: aws
   configBase: memfs://tests/complex.example.com
-  containerRuntime: docker
```

Comment:
Should use `1.20.0` as k8s version in these tests, to see how things actually change.

Reply:
I'm considering updating all of the `options.yaml` under `tests\integration\create_cluster` to use K8s v1.20.1 and running `hack\update-expected.sh` as the K8s versions are inconsistent across the test yaml scripts, but I'm not sure what the overal purpose of the tests is.

Is it to test the creation of a kOps cluster using the earliest supported K8s version or the latest K8s version?

Reply:
This error comes up quite often if I don't update the K8s version with `Networking: calico` option as well.

```
--- FAIL: TestCreateClusterSharedVPC (0.13s)
    create_cluster_integration_test.go:201: error running create cluster: --networking with CNI plugin is required for containerd
```

Reply:
If we stick with the unchanged K8s versions, I can just set `containerRuntime: docker` in the `options.yaml` files to overcome this error.

Reply:
containerd requires setting a CNI like calico, so all tests should have it. Can you check if this fixes things?

Reply:
Yes, I've added that to `tests\integration\create_cluster\complex\options.yaml` and it resolves the issue for this specific test, but should I update all tests to k8s `v1.20.1` or set `ContainerRuntime: docker` for the other tests?

```
Networking: calico
KubernetesVersion: v1.20.1
```

It seems strange to have K8s beta versions e.g. `v1.15.6-beta.1` defined in the integration tests, so I'm considering updating all _create_cluster_ tests to K8s v1.20.1.

Reply:
👍 from me to update all these tests to v1.20.1.

Reply:
thanks for confirming @hakman, I believe it's the correct approach to be testing against the latest K8s version.